opennms-doc/guide-admin/src/asciidoc/text/database-reports/database.adoc

```diff
@@ -264,17 +264,58 @@ To get the appropriate `interface descriptor` depends on the input parameter.
 [[ga-database-reports-https]]
 ==== Use HTTPS
 
-To establish a secure connection to the _Measurements API_ the public certificate of the running _{opennms-product-name}_ must be imported to the _Java Trust Store_.
-In Addition _{opennms-product-name}_ must be configured to use that _Java Trust Store_.
+To establish a secure connection to the _Measurements API_, you must import the public certificate of the running _{opennms-product-name}_ to the _Java Trust Store_.
+In addition, _{opennms-product-name}_ must be configured to use that _Java Trust Store_.
 Please follow the instructions in this <<ga-operation-ssl-opennms-trust-store,chapter>> to setup the _Java Trust Store_ correctly.
 
 In addition please also set the property `org.opennms.netmgt.jasper.measurement.ssl.enable` in `$OPENNMS_HOME\etc\opennms.properties` to `true` to ensure that only secure connections are established.
 
 WARNING: If `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false` an accidentally insecure connection can be established to the _Measurements API_ location.
-         A SSL secured connection can be established even if `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`.
+         An SSL secured connection can be established even if `org.opennms.netmgt.jasper.measurement.ssl.enable` is set to `false`.
 
 [[ga-database-reports-limitations]]
 === Limitations
 
  * Only a _JDBC Datasource_ to the _{opennms-product-name} Database connection_ can be passed to a report, or no datasource at all.
    One does not have to use the datasource, though.
+
+=== Creating PDF Reports from Grafana Dashboards Using OpenNMS
+
+OpenNMS provides three templates to create a PDF report from an existing Grafana dashboard. You can also schedule and email these PDF reports to anyone:
+
+* Keep staff without access to OpenNMS informed about network performance for improved capacity planning
```

Comment:
"OpenNMS" same here, usage of the {opennms-product-name} for OpenNMS Horizon / OpenNMS Meridian.